src/react-skycons.js

```diff
@@ -44,6 +44,9 @@ const ReactSkycons = React.createClass({
    this.state.skycons.set(React.findDOMNode(this), Skycons[nextProps.icon]);
   },
 
+  componentWillUnmount: function componentWillUnmount() {
+    this.state.skycons.remove(React.findDOMNode(this));
```

Comment:
`remove` is the only method in the API: https://github.com/darkskyapp/skycons/blob/master/skycons.js#L684